fft/src/KokkosFFT_Cuda_transform.hpp

```diff
@@ -10,45 +10,47 @@
 
 namespace KokkosFFT {
 namespace Impl {
-template <typename... Args>
-inline void exec_plan(cufftHandle& plan, cufftReal* idata, cufftComplex* odata,
-                      int /*direction*/, Args...) {
-  cufftResult cufft_rt = cufftExecR2C(plan, idata, odata);
+template <typename ScopedPlanType>
+inline void exec_plan(ScopedPlanType& scoped_plan, cufftReal* idata,
+                      cufftComplex* odata, int /*direction*/) {
```

Comment:
Do these functions work with a `ScopedPlanType` different than a `ScopedCufftPlan` ? If not I would remove the template argument.

Reply:
Now using `ScopedCufftPlan` explicitly